.ci/run_tests.sh

```diff
@@ -29,56 +43,70 @@ else
     export TEST_CMD="bin/busted $BUSTED_ARGS,postgres,db"
 fi
 
-if [[ "$KONG_TEST_COVERAGE" = true ]]; then
-    export TEST_CMD="$TEST_CMD --keep-going"
-fi
-
 if [ "$TEST_SUITE" == "integration" ]; then
+    integration_test_file=/tmp/integration_tests.$$.txt
+    trap "rm -f $integration_test_file" 0
+    # We're sorting the list of tests in a random order based on the
+    # pseudo random number generator of bash.  This should level the
+    # run time between the two splits better than performing a static
+    # split based on directory name.  Previously, all 05-proxy tests
+    # were run in the second split, which took three times as long as
+    # running all the other tests.
```

Comment:
I think this random sort needs to be done in a prior build step such that all runners can be made to have the same copy of this file. Otherwise we are not guaranteed to actually run all of the tests.

Reply:
See above.  The shuffle is using a PRNG with a fixed seed to solve this issue.